doc/user/inputs.rst

```diff
@@ -1527,27 +1537,43 @@ Settings Report
 This document lists all the `settings <#the-settings-input-file>`_ in ARMI.  
 
 They are all accessible to developers
-through the :py:class:`armi.settings.caseSettings.Settings` object, which is typically stored in a variable named
-``cs``. Interfaces have access to a simulation's settings through ``self.cs``.
+through the :py:class:`armi.settings.caseSettings.Settings` object, which is typically
+stored in a variable named ``cs``. Interfaces have access to a simulation's settings
+through ``self.cs``.
 
 
 .. exec::
     from armi import settings
     import textwrap
 
+    def looks_like_path(s):
+        """Super quick, not robust, check if a string looks like a file path."""
+        if s.startswith("\\\\") or s.startswith("//"):
+            return True
+        elif s[1:].startswith(":\\")
+            return True
+        return False
+
     subclassTables = {}
     cs = settings.Settings()
+
     # User textwrap to split up long words that mess up the table.
     wrapper = textwrap.TextWrapper(width=25, subsequent_indent='')
     wrapper2 = textwrap.TextWrapper(width=10, subsequent_indent='')
-    content = '\n.. list-table:: ARMI Settings\n   :header-rows: 1\n   :widths: 30 30 10 10\n    \n'
+    content = '\n.. list-table:: ARMI Settings\n   :header-rows: 1\n   :widths: 20 30 15 15\n    \n'
     content += '   * - Name\n     - Description\n     - Default\n     - Options\n'
 
     for setting in sorted(cs.values(), key=lambda s: s.name):
         content += '   * - {}\n'.format(' '.join(wrapper.wrap(setting.name)))
         content += '     - {}\n'.format(' '.join(wrapper.wrap(str(setting.description) or '')))
-        content += '     - {}\n'.format(' '.join(['``{}``'.format(wrapped) for wrapped in wrapper2.wrap(str(getattr(setting, 'default', None)).split("/")[-1])]))
-        content += '     - {}\n'.format(' '.join(['``{}``'.format(wrapped) for wrapped in wrapper.wrap(str(getattr(setting,'options','') or ''))]))
+        default = str(getattr(setting, 'default', None)).split("/")[-1]
+        options = str(getattr(setting,'options','') or '')
+        if looks_like_path(defaults):
```

Comment:
Something is wonky here. I don't see where `defaults` (with the `s`) is defined.